apps/autoscheduler/cost_model_generator.cpp

```diff
@@ -262,8 +262,8 @@ class CostModel : public Generator<CostModel<training>> {
         Expr native_vector_size = schedule_features(n, idx++, w);
         Expr num_vectors = schedule_features(n, idx++, w);
         Expr num_scalars = schedule_features(n, idx++, w);
-        Expr vector_loads_per_vector = schedule_features(n, idx++, w);
         Expr scalar_loads_per_vector = schedule_features(n, idx++, w);
+        Expr vector_loads_per_vector = schedule_features(n, idx++, w);
```

Comment:
LGTM but this is still intrinsically fragile. Would be nice to follow up by changing all these to use explicit enums defined in (say) Featurization.h.